popup.css

```diff
@@ -0,0 +1,142 @@
+.projectPopup {
+  position: fixed;
+  top: 0;
+  z-index: 1;
+  background-color: rgba(193, 199, 208, 0.9);
+  height: 100vh;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.project1 {
+  background-color: #fff;
+  height: 97%;
+  width: 92%;
+  border-radius: 10px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.hide {
+  display: none;
+}
+
+.popupTop {
+  width: 85%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 15px;
+}
```

Comment:
- [ ] It is normal that the modal window exceeds the screen in the y-axis I recommend using `overflow-y: scroll;`. Please 🙏🏽 make sure that the Title and the close button are available for better UX.